src/main/java/com/dev/cinema/model/Movie.java

```diff
@@ -0,0 +1,55 @@
+package com.dev.cinema.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Movie {
```

Comment:
In this case, Hibernate will create a table "Movie" but it is not the best practice to name the table with the first letter in the Upper case. You can use an annotation Table with parameter name